### DISCUSSION TITLE
Add phone icon

### DISCUSSION THREAD

**Discussion by techfg** [withastro/starlight#2725] [Feature Requests]
### What version of `starlight` are you using?

0.30.2

### What is your idea?

Add phone icon from https://iconscout.com/unicons/free-line-icons/phone

![image](https://github.com/user-attachments/assets/b6bab579-f225-4f99-80bc-3faa0ca82c77)

### Why is this feature necessary?

While `email` exists currently, there is no icon to represent a phone number.

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Sure, happy to add this. We probably can’t add it as a social icon option as we don’t currently have a system for translating the labels for those. But adding this as a regular icon would be fine 👍

**Comment by delucis** [comment]
Closed by #2726
